client/src/components/relatedProducts/outfitList.jsx

```diff
@@ -89,11 +89,14 @@ const Outfit = (props) => {
     }
 
   return (
-    <div>
+    <div
+     data-testid = 'outfitCarousel'
+     >
       <span>
```

Comment:
I think it would look a little neater if lines 92-94 is all on one line. That is more just my relative idea of what looks neat, rather than any solid fact to go on, so ultimately do what you think looks best.

Reply:
got it, expected to add more properties here and moved this one to a separate line for readability when that happens but it wasn't necessary yet.